app/src/main/res/layout/sheet_contact_patient.xml

```diff
@@ -3,9 +3,8 @@
   xmlns:tools="http://schemas.android.com/tools"
   android:id="@+id/contentFlipper"
   android:layout_width="match_parent"
-  android:layout_height="match_parent"
-  android:animateLayoutChanges="true"
```

Comment:
`animateLayoutChanges=true` has been removed. Is this intentional?

Reply:
Cool, could we consider showing something manually instead? This is a database call, so some form of transition might be useful.

Reply:
Even with or without the `animateLayoutChanges` we are simply showing the sheet and controls inside it. The only difference is a slight fade-in for the overdue controls. For that, I can look into `TransitionManager` usage in a different PR. Is that fine?

Reply:
Yeah, sounds good.

Reply:
Resolved it in #2531